knot-thing/src/main/java/com/cesar/knot_sdk/knot_messages/KNoTMessageDataItem.kt

```diff
@@ -0,0 +1,10 @@
+package com.cesar.knot_sdk.knot_messages
+
+/**
+ * This is the class that is serialized for one data value, from one sensor.
+ * @property id the Thing unique identifier
+ * @property data the data batch that is to be deployed
```

Comment:
I got what you mean here but I'd change from `deployed` to `sent`. At least for me, it becomes clearer.

Reply:
What do you think about it?